yarn-project/cli/src/cmds/l1/advance_epoch.ts

```diff
@@ -0,0 +1,12 @@
+import { CheatCodes, createPXEClient, makeFetch } from '@aztec/aztec.js';
+import { type LogFn } from '@aztec/foundation/log';
+
+export async function advanceEpoch(l1RpcUrl: string, rpcUrl: string, log: LogFn) {
+  const pxe = createPXEClient(rpcUrl, makeFetch([], true));
+  const rollupAddress = await pxe.getNodeInfo().then(i => i.l1ContractAddresses.rollupAddress);
```

Comment:
Is there no way to get the address without instantiating the entire pxe?

Reply:
Yep, we could accept it as an argument, or ask the node directly. I built it this way for consistency with the other commands, but we should allow these other options as you say.